.gitignore

```diff
@@ -2,6 +2,7 @@
 #################################
 /nbproject
 .idea
+atlassian-ide-plugin.xml
```

Comment:
That's not necessary as it is only relevant for PhpStorm and only for people with this plugin. Also as of the newest version of this plugin, the file is stored in .idea anyway.